### ISSUE TITLE
thread 'main' panicked at 'byte index 11 is not a char boundary

### ISSUE THREAD

**Issue by einsli** [Bluemi/musicus_rs#1]
When there is Chinese in the same level directory of the parent directory of the project or a file in the absolute path, an error will be reported!
In this case there will be such an error：
eg. the project path is /home/user/Documents/Rust-Learn/musicus_rs, but in the path of  /home/user/Documents/, there is a file or a directory named by Chinese, when I jsut run the command ```rustup run nightly cargo run```, it can cause the error

**Comment by Bluemi** [comment]
Hey, thanks for the issue report.

Unfortunately, I cannot reproduce the problem. When I start with the following folder structure, no error occurs:
```
/home/user/documents/
├── musicus_rs
│   ├── Cargo.toml
│   ├── src
│   │   └── ...
│   └── 汉字
│       └── 汉字.wav
└── 汉字

```

Can you send a backtrace of the crash? As ncurses prevents normal backtrace output you can create it by running the following command in a bash shell:
```bash
RUST_BACKTRACE=1 cargo run --release 2> error_log.txt
```

I want to mention that this project is still in early development, so there are probably some other problems.

**Comment by einsli** [comment]
Thank you very much for your reply. Sorry for taking so long to reply to you. Due to network reasons, I cannot log in for a while. In response to the problem I submitted, I tested it again and it worked normally. Maybe I was willing to operate at the beginning. In addition, I am very interested in your project. Is there a special discussion group for me to join? If I can enter, I am very grateful!

**Comment by Bluemi** [comment]
Glad to see it working now.

I have enabled discussions for this repository in case you or anyone else wants to share ideas or ask questions. Feel free to participate :)

**Comment by Bluemi** [comment]
This issue was already closed, but I was able to reproduce and fix in 15710fa8d2269b6b6ef308563fabc37eb97ecf3f
